lib/kitchen/directions/move_solutions_to_answer_key/answer_key_cleaner.rb

```diff
@@ -12,7 +12,8 @@ def bake(book:)
       answer_key_chapters = book.search(
         '.os-eob[data-type="composite-chapter"] > [data-type="composite-page"]')
       answer_key_chapters.each do |container|
-        container.trash unless container.contains?('[data-type="solution"]')
+        container.trash unless container.contains?('[data-type="solution"]') ||
+                               container.contains?('[data-type="question-solution"]')
```

Comment:
i think `'[data-type="solution"], data-type="question-solution"]'` will work as a selector that can grab both of them, if u wanna condense